plotly/tools.py

```diff
@@ -2274,6 +2310,160 @@ def create_candlestick(open, high, low, close,
 
         return dict(data=data, layout=layout)
 
+    @staticmethod
+    def create_distplot(hist_data, group_labels,
+                        bin_size=1., curve_type='kde',
+                        show_hist=True, show_curve=True,
+                        show_rug=True, **kwargs):
+        """
+        BETA function that creates a distplot similar to seaborn.distplot
+
+        The distplot can be composed of all or any combination of the following
+        3 components: (1) histogram, (2) curve: (a) kernal density estimation
+        or (b)normal curve, and (3)rug plot. Additionally, multiple distplots
+        (from multiple datasets) can be created in the same plot.
+
+        :param (list) hist_data: list of histogram data, should be a list of
+            lists if multiple data sets are plotted on the same plot
+        :param (list) group_labels: list of strings of the name of each data
```

Comment:
One more note (man, you guys probably think I'm so nit-picky :p)

Our style guide for JS documentation and Python in streambed follow similar conventions, *JS Docs* for javascript and *reST* for Python:

* [JS](https://github.com/plotly/dev-docs/blob/master/reference/styleguides/javascript.md#comments)
* [Python](https://github.com/plotly/dev-docs/blob/master/reference/styleguides/python.md#function-docstrings)

Not that that's held as a commandment by the team, but seeing consistency in our code is pretty nice, imo.